### ISSUE TITLE
differentiating deployment target

### ISSUE THREAD

**Issue by gambtho** [JeroenSoeters/wheel#11]

**Comment by krogebry** [comment]
Is this the same as putting a version string in the StackName?

**Comment by JeroenSoeters** [comment]
Nope, this is providing target cloud aws/dedicated-hw/azure/etc and the config for that cloud. Initially we'll support AWS and dedicated hw once the [client I cannot mention publicly] project picks up again.
